src/hotspot/share/gc/shenandoah/shenandoahNumberSeq.cpp

```diff
@@ -56,7 +56,7 @@ void HdrSeq::add(double val) {
   int mag;
   if (v > 0) {
     mag = 0;
-    while (v > 1) {
```

Comment:
This is a bug fix that has been independently pushed to tip. You can ignore it and it'll find its way into shenandoah in due course.